packages/client/src/api/wargames_api/index.ts

```diff
@@ -159,9 +160,12 @@ export const listenForWargameChanges = (name: string, dispatch: PlayerUiDispatch
   listenNewMessage({ db, name, dispatch })
 }
 
-export const pingServer = (): Promise<any> => {
+export const pingServer = (activityDetails: {wargame: string, role: string}): Promise<any> => {
   const activityTime = expiredStorage.getItem(ACTIVITY_TIME) || 'The player has not shown any activity yet'
-  return fetch(serverPath + 'healthcheck/' + activityTime + '/healthcheck', {
+  const activityType = expiredStorage.getItem(ACTIVITY_TYPE) || 'The player has not shown any activity yet'
+  const activityLogs: ActivityLogsInterface = { activityTime, activityType, ...activityDetails }
+
+  return fetch(serverPath + 'healthcheck/' + JSON.stringify(activityLogs) + '/healthcheck', {
```

Comment:
@lilitkarapetyan - would it be a problem if we passed the data as four fields , instead of using `stringify`?  The use of `stringify` is making the call harder to read:
![image](https://user-images.githubusercontent.com/1108513/153588473-6d335c2a-c79f-4cf3-bd50-2f5c41bccad3.png)

There may be a strong reason to keep `stringify`.  If there is, please just say.